test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2726,6 +2726,38 @@ def forward(self, input, indices):
         indices = torch.tensor([[1, 0], [0, 1], [0, 1]], dtype=torch.int64)
         self.run_test(GatherModel(), input=(input, indices))
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_gather_constant_fold(self):
+        class GatherModule(torch.nn.Module):
+            def __init__(self):
+                super(GatherModule, self).__init__()
+                self.register_buffer("weight", torch.ones(5))
+
+            def forward(self, x):
+                # shape is of rank 0
+                shape = self.weight.shape[0]
```

Comment:
I don't see a torch gather there. Is this `shape[0]` converting to onnx Gather?

Reply:
yes shape[0] is converting to Gather op